packages/lesswrong/lib/collections/featuredResources/schema.ts

```diff
@@ -1,27 +1,31 @@
 const schema: SchemaType<DbFeaturedResource> = {
   title: {
     type: String,
+    nullable: false,
```

Comment:
Pretty sure this entire collection is EA-forum specific, and needs review by them